app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
Using a dictionary with names as keys (`Person.people`) can lead to issues if two people have the same name, as one entry will overwrite the other. Consider using unique identifiers for each person.